ports.js

```diff
@@ -1 +1,57 @@
-app = Elm.Main.fullscreen();
+const KINTO_SERVER = "https://kinto.dev.mozaws.net/v1";  // No trailing slash
+const KINTO_BUCKET = "stepfunction";
+const KINTO_COLLECTION = "manual_steps";
```

Comment:
Actually this is a review of @n1k0 that asked me that have the settings all in one place. If you see how we can calculate the redirect_uri in Elm I would be glad to move it there.